docs/website/blog/2024-03-25-reverse_etl_dlt.md

```diff
@@ -0,0 +1,196 @@
+---
+slug: reverse-etl-dlt
+title: "dlt adds Reverse ETL - build a custom destination in minutes"
+image:  https://storage.googleapis.com/dlt-blog-images/reverse-etl.png
+authors:
+  name: Adrian Brudaru
+  title: Open source Data Engineer
+  url: https://github.com/adrianbr
+  image_url: https://avatars.githubusercontent.com/u/5762770?v=4
+tags: [reverse etl, pythonic]
+---
+
+# Pythonic reverse ETL is here
+
+## Why python is the right approach for doing Reverse ETL
+
+Reverse ETL is generally about putting data into a business app. This data would often come from a SQL database used as a middle layer for data integrations and calculations.
```

Comment:
As someone unsure about what reverse ETL does, the opening doesn't clarify this for me. I did a simple Google search and the below is a bit clearer explanation

> In a nutshell, reverse ETL is a technology for taking cleaned and processed data from the data warehouse and ingesting it back into business applications such as Salesforce, which can be used for business operations and forecasting.